src/js/containers/AnalyticsPage/AnaltyicsPage.js

```diff
@@ -281,7 +282,22 @@ class AnaltyicsPage extends Component {
                             </LayoutContainer> 
                         </TabsPanel>
                     </Tabs>
-                    </div>
+                  
+                    <LayoutRow>
+                      <LayoutColumn width={31/64}>
+                        <AnalyticCard title={ formatMessage({ id: 'container.analyticCard.ssl.title' }) } description={ formatMessage({ id: 'container.analyticCard.duration' }) } data={ analytics.totals.requests.ssl } dataType={ formatMessage({ id: 'container.analyticCard.ssl.datatype' }) } />
+                      </LayoutColumn>
+                      <LayoutColumn width={2/64}>&nbsp;</LayoutColumn>
+                      <LayoutColumn width={31/64}>
+                        <AnalyticCard title={ formatMessage({ id: 'container.analyticCard.bandwidth.title' }) } description={ formatMessage({ id: 'container.analyticCard.duration' }) } data={ analytics.totals.bandwidth } dataType={ formatMessage({ id: 'container.analyticCard.bandwidth.datatype' }) } />
+                      </LayoutColumn>
+                    </LayoutRow>
+
+                    <LayoutRow>
```

Comment:
Why not have the last row be `width={32/65}` and remove this empty `<LayoutRow/>`?